### ISSUE TITLE
Warmup dataloader batch size double

### ISSUE THREAD

**Issue by lfb-1** [LiJunnan1992/DivideMix#4]
Hi,
I wonder why in the warmup process, the dataloader batch size has to be doubled? Can I use the normal batch size?

Cheers

**Comment by LiJunnan1992** [comment]
Yes you can totally use normal batch size. I double it just to speed up training.
